executor/executor.go

```diff
@@ -21,7 +23,10 @@ func (e ErrStatus) Error() string {
 }
 
 // Run executes a slice of CommandDefs
-func Run(output io.Writer, job screwdriver.JobDef) error {
+func Run(path string, output io.Writer, job screwdriver.JobDef) error {
+	if err := chdir(path); err != nil {
```

Comment:
So does that prevent people from changing the directory in step 1, then using that new dir in step 2?